SU2_CFD/src/integration/CMultiGridIntegration.cpp

```diff
@@ -688,6 +688,25 @@ void CMultiGridIntegration::NonDimensional_Parameters(CGeometry **geometry, CSol
       solver_container[FinestMesh][FLOW_SOL]->Momentum_Forces(geometry[FinestMesh], config);
       solver_container[FinestMesh][FLOW_SOL]->Friction_Forces(geometry[FinestMesh], config);
 
+      /*--- Calculate the turbo performance ---*/
```

Comment:
This is just food for thought, but I have never really liked that we compute the output quantities in this location. Seems like we could have a better design to make it more general. Just a TODO for the future.